examples/batch_logging.cc

```diff
@@ -0,0 +1,181 @@
+        runnables {
+          script {
+            text: "echo Hello world! This is task ${BATCH_TASK_INDEX}. This job has a total of ${BATCH_TASK_COUNT} tasks."
+          }
+        }
+      }
+    }
+    allocation_policy {
+      instances {
+        policy { machine_type: "e2-standard-4" provisioning_model: STANDARD }
+      }
+    }
+    labels { key: "env" value: "testing" }
+    labels { key: "type" value: "script" }
+    logs_policy { destination: CLOUD_LOGGING }
+  )pb";
+  auto* job = request.mutable_job();
+  if (!google::protobuf::TextFormat::ParseFromString(kText, job)) {
+    throw std::runtime_error("Error parsing Job description");
+  }
+  // Create a client and issue the request.
+  return client.CreateJob(request).value();
+}
+
+void WaitForJob(google::cloud::batch_v1::BatchServiceClient client,
+                std::string job_name) {
+  std::cout << "\nWaiting for " << job_name << std::flush;
+  // It taskes about 60 seconds to finish a job, so waiting for about 5 minutes
+  // seems enough.
+  auto const polling_period = std::chrono::seconds(10);
+  for (int i = 0; i != 30; ++i) {
+    auto response = client.GetJob(job_name);
+    auto done = [](auto state) {
+      using google::cloud::batch::v1::JobStatus;
+      return state == JobStatus::SUCCEEDED || state == JobStatus::FAILED;
+    };
+    if (response && done(response->status().state())) {
+      std::cout << ".DONE\n";
+      return;
+    }
+    std::cout << "." << std::flush;
+    std::this_thread::sleep_for(polling_period);
+  }
+  std::cout << ".DONE (TIMEOUT)\n";
+  throw std::runtime_error("Timeout waiting for job");
+}
+
+void AutoRun(std::vector<std::string> const& argv) {
+  namespace examples = ::google::cloud::testing_util;
+  using ::google::cloud::internal::GetEnv;
+  if (!argv.empty()) throw examples::Usage{"auto"};
+  examples::CheckEnvironmentVariablesAreSet({
+      "GOOGLE_CLOUD_PROJECT",
+      "GOOGLE_CLOUD_CPP_TEST_REGION",
+  });
+  auto const project_id = GetEnv("GOOGLE_CLOUD_PROJECT").value();
+  auto const location_id = GetEnv("GOOGLE_CLOUD_CPP_TEST_REGION").value();
+
+  auto generator = google::cloud::internal::DefaultPRNG(std::random_device{}());
+  auto const job_id =
+      kJobPrefix + google::cloud::internal::Sample(
+                       generator, 32, "abcdefghijklmnopqrstuvwxyz");
+
+  auto client = google::cloud::batch_v1::BatchServiceClient(
+      google::cloud::batch_v1::MakeBatchServiceConnection());
+
+  // Create the job to drive this test.
+  auto job = CreateTestJob(client, project_id, location_id, job_id);
+  std::cout << "Created test job: " << job.name() << "\n";
+
+  // Wait until the job completes, otherwise the logs may be empty. The logs may
+  // still be delayed, but this is less likely.
+  WaitForJob(client, job.name());
+
+  std::cout << "Running JobLogs() test" << std::endl;
+  JobLogs({project_id, location_id, job_id});
+
+  //  (void) client.DeleteJob(job.name()).get().value();
```

Comment:
Commented out?

Reply:
Fixed.  I needed to debug something and forgot to restore this.